### DISCUSSION TITLE
scores not uploading to the server

### DISCUSSION THREAD

**Discussion by ignatgeimer** [ppy/osu#30542] [Q&A] [answered]
### Type

Game behaviour

### Bug description

when i starting any map this notification pops up, im playing on newest version, tried reinstalling, turning off anti-malware programms, and checked firewall.

### Screenshots or videos

![image](https://github.com/user-attachments/assets/ad159318-bfa2-4b12-8b7d-9635a8416189)

### Version

2024.1009.1

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17664337/compressed-logs.zip)

**Comment by smoogipoo** [comment]
Please zip up the osu! folder (`%LOCALAPPDATA%/osulazer`) and send it to `pe@ppy.sh` with a link back to this thread.

**Comment by smoogipoo** [comment, marked as answer]
Some of the files in your installation are corrupted. I suggest you delete that directory and re-install the game.

You won't lose any data doing this.

**Comment by ignatgeimer** [comment]
worked, thank you!
